app/controllers/registrations_controller.rb

```diff
@@ -14,6 +14,7 @@ def create
 
     if @registration.valid?
       update_session_registration_hash
+      candidate_submission if step_name.include?("accept_privacy_policy")
```

Comment:
Can we rename this to make it clearer? so something like `perform_candidate_submission`. Maybe abstract `step_name.include?("...")` to be under a `last_step` method as well, then it would read:

```
perform_candidate_submission if last_step?
```

Reply:
Will do, although it is refactoring to add more lines 😀